src/sparseml/yolov5/README.md

```diff
@@ -0,0 +1,126 @@
+# SparseML YOLOv5 Integration
+This directory combines the SparseML recipe-driven approach with the 
+[ultralytics/yolov5](https://github.com/ultralytics/yolov5) repository.
+By integrating the robust training flows in the `yolov5` repository with the SparseML code base, we enable model sparsification techniques on the popular [YOLOv5 architecture](https://github.com/ultralytics/yolov5/issues/280)
+as well as the updated [YOLOV5x6 architecture](https://github.com/ultralytics/yolov5/releases/tag/v5.0),
+creating smaller and faster deployable versions.
+The techniques include, but are not limited to:
+- Pruning
+- Quantization
+- Pruning + Quantization
+- Sparse Transfer Learning
+
+After training, the model can be deployed with Neural Magic's DeepSparse Engine. The engine enables inference with GPU-class performance directly on your CPU.
+
+This integration enables spinning up one of the following end-to-end functionalities:
+- **Sparsification of YOLOv5 Models** - easily sparsify any of the YOLOV5 and YOLOV5x6 models, from YOLOv5n to YOLOv5x models. 
+- **Sparse Transfer Learning** - fine-tune a sparse backbone model (or use one of our [sparse pre-trained models](https://sparsezoo.neuralmagic.com/?domain=cv&sub_domain=detection&page=1)) on your own, private dataset.
+
+## Installation
+
+```pip install sparseml[torchvision]```
+
+Note: YOLOV5 will not immediately install with this command. Instead, a sparsification-compatible version of YOLOv5 will install on the first invocation of the YOLOv5 code in SparseML.
+
+## Tutorials
+
+- [Sparsifying YOLOv5 Using Recipes](https://github.com/neuralmagic/sparseml/blob/main/integrations/ultralytics-yolov5/tutorials/sparsifying_yolov5_using_recipes.md)
+- [Sparse Transfer Learning With YOLOv5](https://github.com/neuralmagic/sparseml/blob/main/integrations/ultralytics-yolov5/tutorials/yolov5_sparse_transfer_learning.md)
+
+## Quick Tour
+
+### Recipes
+
+Recipes encode the instructions and hyperparameters for sparsifying a model using modifiers to the training process.
+The modifiers can range from pruning and quantization to learning rate and weight decay.
+When appropriately combined, it becomes possible to create highly sparse and accurate models.
+
+This integration adds a `--recipe` argument to the [`train.py` script](https://github.com/neuralmagic/yolov5/blob/master/train.py).
+The argument loads an appropriate recipe while preserving the rest of the training pipeline.
+Popular recipes used with this argument are found in the [`recipes` folder](./recipes).
```

Comment:
Noting: Could you confirm if relative links work with Sphinx? I remember @jeanniefinks noticed they weren't